### ISSUE TITLE
Add support for hashtags

### ISSUE THREAD

**Issue by alimony** [bluesky-social/social-app#848]
Maybe I’m stating the obvious, but general support for hashtags would be really nice. Even in its simplest form (allow search for them + turn them into links in posts) it would be very valuable.

**Comment by Luke717** [comment]
I agree, hashtag support would be very useful precisely for the reasons you point out. Support wouldn't have to be as [detailed or as complex](https://business.twitter.com/en/blog/how-to-create-and-use-hashtags.html) as it is on, say, Twitter, just the simplest functionality mentioned to start with. It really would be a valuable feature.

**Comment by muchmich** [comment]
I would like to suggest hashtags being "outside" of the post, more similar to tumblr's tagging system.
Something like this:
![image](https://github.com/bluesky-social/social-app/assets/120081638/e51f1e78-ebc9-44a4-a867-0dac47a05c7c)
![image](https://github.com/bluesky-social/social-app/assets/120081638/73b2a176-a80c-4c45-9d97-8dfc74e48d56)

**Comment by simonetthomas** [comment]
Out of curiosity : when it's implemented, will all the "hashtags" in existing posts turned into real clickable hashags ? Or would this only affect new posts ?

**Comment by Sininini** [comment]
I would prefer if all the existing "hashtags" got turned into real ones. People are already using them and it would be nice to not have to repost art and stuff just to redo the hashtags.

**Comment by danorton** [comment]
The ability to “follow” hashtags would also be helpful.

**Comment by 8ullyMaguire** [comment]
https://github.com/bluesky-social/social-app/issues/1534

**Comment by 8ullyMaguire** [comment]
Related to #1533: Comprehensive Tagging System.

Benefits of a comprehensive tagging system over hashtags:
- Filter sensitive content - Users can avoid content about phobias, traumatic events, etc. that they don't want to see
- More flexible searching - Combine, include, or exclude any tags
- Enhanced organization - Logical categorization for easier discovery
- Improved discovery - Browse and search tags to find relevant content
- Separate from content - Tags are non-disruptive metadata, hashtags are just spam
- Post-publish tagging - Improve organization after posting
- Granular feed curation - Highly customized feeds by tags

**Comment by RBeerDevelopment** [comment]
Just wanted to push this one again. Hashtag support would really be great and at least in my opinion having a minimal version first would be great. Extended features like having full tagging support could still be added in the future. 
I'm also open to work on this though I fully understand if this is something that the core team wants to handle

**Comment by Zero3K** [comment]
The eric/tags branch contains the work being done on implementing them.

**Comment by mschwendener** [comment]
This can be closed, no?

**Comment by Sininini** [comment]
They still haven’t added more than bare minimum for hashtags support. I personally would like to finally have outside tags support in the main app
Should this be closed and a new issue about the missing features created?

**Comment by anentropic** [comment]
I would like to be able to treat a hashtag exactly like a feed, specifically to be able to follow it

**Comment by gitcos** [comment]
> They still haven’t added more than bare minimum for hashtags support. I personally would like to finally have outside tags support in the main app Should this be closed and a new issue about the missing features created?

I think you're right, they have added support for hashtags, though at a bare minimum and not a way that's really useful.

I've written a new feature request that's more specific about how they could support *useful* tags.  Please click the thumbs up on that one if you voted for this one.

https://github.com/bluesky-social/social-app/issues/7693
